src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const Add = 'Add';
+const Delete = 'Delete';
```

Comment:
- [ ] Kindly Define ADD_BOOK and REMOVE_BOOK as action types. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`